adapters/adapter.go

```diff
@@ -9,32 +9,42 @@ import (
 	"time"
 )
 
+// Adapters connect prebid-server to a demand partner. Their primary purpose is to produce bids
+// in response to Auction requests.
 type Adapter interface {
+	// A name which uniquely identifies this adapter.
+	// This must not overlap with any other adapters in prebid-server.
 	Name() string
 	FamilyName() string
 	GetUsersyncInfo() *pbs.UsersyncInfo
```

Comment:
Returns the parameters so that we can do a cookie sync between the adapter and prebid server